src/app.js

```diff
@@ -1 +1,66 @@
-console.log('Hallo daar!');
\ No newline at end of file
+console.log('Hallo daar!');
+
+import axios from 'axios';
+
+const list = document.getElementById( "list-of-countries" );
+
+async function fetchData() {
+    const URI = 'https://restcountries.com/v2/all';
+
+    try {
+        const result = await axios.get(URI);
+        console.log(result.data);
+
+        result.data.sort ((a,b) => (a.population - b.population));
+
+        list.replaceChildren();
```

Comment:
Volgens mij kan je deze methode weglaten aangezien er geen children in het ul-element staan